scripts/windows/build.bat

```diff
@@ -35,14 +35,14 @@ FOR /f "tokens=1-2" %%i in ('python --version') do (
 )
 @echo Installed python version: %PYTHON_VERSION%
 
-SET ERRORLEVEL=
+CALL :clear_error
 WHERE /Q pip
 IF ERRORLEVEL 1 (
         ECHO pip wasn't found. Attempting to install...
         wget -O .\dep\get-pip.py https://bootstrap.pypa.io/get-pip.py
         python  .\dep\get-pip.py
-	SET ERRORLEVEL=
-	WHERE /Q pip
+		CALL :clear_error
```

Comment:
Shouldn't this remain in the same indentation?

Reply:
Indentation was mixed, thanks for catching. Will apply common formatting.